### DISCUSSION TITLE
How do you teleport enemies to a player?

### DISCUSSION THREAD

**Discussion by ivjvon** [winstxnhdw/lc-hax#282] [Q&A]
Possible?

**Comment by winstxnhdw** [comment]
use the `mob` command

**Reply by ivjvon** [reply]
It says no mobs near even tho there are that have already spawned

**Reply by winstxnhdw** [reply]
It means there’s no teleportable mobs. Not all mobs can be teleported because they can bug your game out.

**Reply by winstxnhdw** [reply]
Nevermind, seems like there was a bug. I fixed this in latest.
